ReactAndroid/src/main/java/com/facebook/react/uimanager/ReactAccessibilityDelegate.java

```diff
@@ -123,20 +125,32 @@ public enum AccessibilityRole {
     TIMER,
     LIST,
     GRID,
+    PAGER,
+    SCROLLVIEW,
+    HORIZONTALSCROLLVIEW,
+    VIEWGROUP,
+    WEBVIEW,
+    DRAWERLAYOUT,
+    SLIDINGDRAWER,
+    ICONMENU,
     TOOLBAR;
 
     public static String getValue(AccessibilityRole role) {
       switch (role) {
         case BUTTON:
           return "android.widget.Button";
+        case DROPDOWNLIST:
+          return "android.widget.Spinner";
+        case EDITTEXT:
+          return "android.widget.EditText";
```

Comment:
TextInput does not need an EditText role on Android. Adding the EditText role removes TalkBack announcements.

The relevant logic is in https://github.com/aosp-mirror/platform_frameworks_base/blob/e648d3cb91f9a0156c729ca18ec27e71f59f9ce2/core/java/android/widget/TextView.java#L12291

https://user-images.githubusercontent.com/24992535/187655686-f6bac081-dd6b-4534-bd02-a11a5bcf27c8.mp4